### PR TITLE
Upgrade to Eclipse Collections 8.2.0.

### PR DESCRIPTION
**Pull request by donraab**
Signed-off-by: Donald Raab <Donald.Raab@bnymellon.com>

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,26 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="EntryPointsManager">
```
Comment (nikhilnanivadekar) [inline .idea/misc.xml]: You need not commit changes to this file. Can you please remove it from the commit?

**Comment by donraab** [comment]
@nikhilnanivadekar should be fixed now.

File: company-kata/company-kata.iml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module external.linked.project.id=":company-kata" org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8" inherit-compiler-output="false">
```
Comment (kristeno) [inline company-kata/company-kata.iml]: what's this?

Comment (donraab) [inline company-kata/company-kata.iml]: Good question.  I think it is something the latest version of IntelliJ has added but honestly I am not sure what it is for.  I will do some research to see if there is anything I should change here.

**Pull request merged**